src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -82,6 +82,7 @@ for further details.
 }
 
 .sidebarContentWrapper {
+  height: 100%; /* for positioning the version switcher */
```

Comment:
❓  was this intended to be included in this PR?

Reply:
Yes!